hermes/bin/src/wasm/module.rs

```diff
@@ -107,6 +107,109 @@ impl Module {
     }
 }
 
+#[cfg(feature = "bench")]
+#[allow(dead_code)]
```

Comment:
Just me nitpicking: is this really needed?

Reply:
yea, actually it is not needed here. Thank you ! 😊